WatsonDeveloperCloud.xcodeproj/project.pbxproj

```diff
@@ -138,6 +152,7 @@
 		7A0F95A91C23288E004C01C0 = {
 			isa = PBXGroup;
 			children = (
+				B1F21DEF1CE2461800393146 /* ToneAnalyzer */,
```

Comment:
There needs to be a major version in the naming such as DialogV1 so ToneAnalyzerV3